pom.xml

```
@@ -128,7 +128,7 @@ under the License.
     <version.maven-shade-plugin>3.5.1</version.maven-shade-plugin>
     <version.maven-site-plugin>3.12.1</version.maven-site-plugin>
     <version.maven-source-plugin>3.3.0</version.maven-source-plugin>
-    <version.maven-surefire>${surefire.version}</version.maven-surefire>
+    <version.maven-surefire>3.2.3</version.maven-surefire>
```

The surefire.version property should be modified instead so it applies to failsafe as well.

```suggestion
    <version.maven-surefire>${surefire.version}</version.maven-surefire>
```